### ISSUE TITLE
"How to configure your learning rate schedule" tutorial

### ISSUE THREAD

**Issue by LukeWood** [keras-team/keras-io#948]
It would be really useful to have a tutorial on configuring learning rate schedule.  Right now, you basically dig around on the internet and look for schedules and try them.  Even if this is actually the optimal way to tune your learning rate schedule, it would be great to have some docs to tell beginners how to go about this.

**Comment by SuryanarayanaY** [comment]
Hi @LukeWood ,

I think the documentation have enough information about custom learning rate scheduler [here](https://keras.io/api/callbacks/learning_rate_scheduler/).

You can define your own learning rate function(eg. scheduler) and then pass it to the **LearningRateScheduler** callback `callback  = tf.keras.callbacks.LearningRateScheduler(scheduler)` and then this **callback** will be passed to `model.fit(...,callbacks=[callback])`. All the required information mentioned in the attached documentation source.

Thanks!
